skopt/acquisition.py

```diff
@@ -4,20 +4,20 @@
 from scipy.stats import norm
 
 
-def gaussian_acquisition_1D(X, model, y_opt=None, acq_func="LCB", xi=0.01,
-                            kappa=1.96):
+def gaussian_acquisition_1D(X, model, y_opt=None, acq_func="LCB",
+                            acq_func_kwargs=None):
     """
     A wrapper around the acquisition function that is called by fmin_l_bfgs_b.
 
     This is because lbfgs allows only 1-D input.
     """
     return _gaussian_acquisition(np.expand_dims(X, axis=0),
-                                 model, y_opt, acq_func, xi, kappa,
+                                 model, y_opt, acq_func_kwargs=acq_func_kwargs,
```

Comment:
It would be nice to keep the explicit arguments here and call the acq function with `func(**kwargs)`. Was thinking we could then send users here from the docs of `Optimizer` with "Additional kwargs, see your acq func's documentation for what is available"